recipe/run_test.sh

```diff
@@ -16,13 +16,13 @@ else
   echo Error
 fi
 
-# limit CPUs in use on PPC64LE, fork() issues
+# limit CPUs in use on PPC64LE and AARCH64, fork() issues
 # occur on high core count systems
 archstr=`uname -m`
 if [[ "$archstr" == 'ppc64le' ]]; then
     TEST_NPROCS=1
 elif [[ "$archstr" == 'aarch64' ]]; then
-    TEST_NPROCS=4
+    TEST_NPROCS=1
```

Comment:
I think this change was made when initially debugging the issue on AArch64 that turned out to require the consecutive registers patch in LLVM - does this need to change back, or can it be left as-is?

Reply:
Sure let's give that a try. Can always revert if needed

```suggestion
    TEST_NPROCS=4
```